.azure/pipelines.yml

```diff
@@ -258,7 +258,7 @@ stages:
         condition: startsWith(variables.image, 'windows')
 
       - bash: |
-          mamba create --quiet --name wheelEnvironment --channel conda-forge python=3.10 mamba
+          conda create --quiet --name wheelEnvironment --channel conda-forge python=3.10
```

Comment:
Why back to conda?

Reply:
Because conda now uses libmamba (or at least can be configured to do so, which I do)